appcues/src/main/java/com/appcues/ui/primitive/ButtonPrimitive.kt

```diff
@@ -28,16 +28,23 @@ internal fun ButtonPrimitive.Compose(modifier: Modifier) {
         color = Color.Transparent,
     ) {
         ProvideTextStyle(style.getTextStyle(LocalContext.current, isSystemInDarkTheme())) {
-            Row(
-                horizontalArrangement = Arrangement.Center,
-                verticalAlignment = Alignment.CenterVertically,
+            Box(
+                contentAlignment = content.style.getBoxAlignment(),
```

Comment:
nice catch here. I left center by default but this is in align with convo from last week